requirements.txt

```diff
@@ -1,11 +1,10 @@
 lxml==4.2.1
 JSON_minify==0.3.0
-torch==0.4.0
 gensim==3.4.0
 tqdm==4.23.3
 numpy==1.14.3
 termcolor==1.1.0
 scikit_learn==0.19.1
-# temporary using version from github due to security issues
```

Comment:
Has this been solved already in the pip release of pyyaml?

Reply:
I misread the security report. Will revert.